book/templating.rst

```diff
@@ -1037,7 +1037,7 @@ argument (or the ``version`` argument) to the desired version:
 
         <img src="<?php echo $view['assets']->getUrl('images/logo.png', null, false, '3.0') ?>" alt="Symfony!" />
```

Comment:
Also I think it might be a good idea to remove horizontal scrollbar on examples, but as I was not sure at which points it would be best to cut I left them unchanged